src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -313,17 +314,14 @@ public void Dispose()
             // Protected implementation of Dispose pattern.
             protected virtual void Dispose(bool disposing)
             {
-                if (disposing)
+                if (disposing && !_isDisposed)
                 {
-                    // Free any other managed objects here.
-                    //
+                    _isDisposed = true;
                     _abortCts.Cancel();
                     _abortCts.Dispose();
                     _abortRegistration.Dispose();
                 }
 
-                // Free any unmanaged objects here.
-                //
                 _fetchResponse?.Dispose();
                 _abortController?.Dispose();
```

Comment:
Should it instead be:
```C#
if (_isDisposed)
{
    return;
}
```
at the beginning, before the `disposing` check, so that we don't bother disposing of these two fields if Dispose was already called?